dasharo-stability/capsule-update.robot

```diff
@@ -104,6 +105,25 @@ CUP150.001 Capsule Update
     Should Contain    ${out}    CapsuleMax
     Should Not Contain    ${out}    CapsuleLast
 
+CUP150.002 Capsule Update Progress Bar - Dasharo Logo
+    [Documentation]    Verify that the Capsule Update screen looks as expected
+    ...    and the progress bar is scaled properly using the default Dasharo
+    ...    logo.
+    # Ensure we're running FW with the default logo
+    Flash Firmware If Not QEMU    default
+    Boot Into UEFI Shell
+    Perform Capsule Update    max_fw_ver.cap
+    Check The Update Screen For The Correct UX
+
+CUP150.002 Capsule Update Progress Bar - Custom Logo
```

Comment:
This could be done as part of `CUP150.001` to save time on flashing the firmware (twice - once for initial image, and then for update). When done as a separate test case here, it would also break the following tests as the flashing would get rid of any customization done after initial flashing performed in suite setup.

Reply:
[Done](https://github.com/Dasharo/open-source-firmware-validation/pull/520/commits/af7dcc6dbc6f01376fd395e21c92c4908925eadb#diff-e5457f1e95f583456d0728805e1a4d16e63924b5c7f79a4ff0bbdcfed0751b7bR98)